web/src/urls.ts

```diff
@@ -34,4 +34,8 @@ export default Object.freeze({
     'https://discourse.mozilla.org/t/help-create-common-voices-first-target-segment/59587',
   TARGET_SEGMENT_INFO_ES:
     'https://discourse.mozilla.org/t/ayuda-a-crear-el-primer-objetivo-segmentado-de-common-voice/60472/',
+
+  HTTP_ROOT: 'https://commonvoice.mozilla.org',
```

Comment:
this should be an array to allow us using voice.m.o and commonvoice.m.o without a banner.

The problem using only one is that the deployment of the new image and the redirection have to be done exactly at the same time, otherwise users will see the banner.
Flux will deploy both changes at once, however there's a delay while the new container is pulled, so for some seconds the redirection will be applied, and the new code won't be running.